app/models/dimensions/item.rb

```diff
@@ -8,6 +8,10 @@ class Dimensions::Item < ApplicationRecord
     where('updated_at < ?', date).where(outdated: true)
   end
 
+  scope :by_natural_key, ->(content_id:, locale:) do
+    where(content_id: content_id, locale: locale, latest: true)
```

Comment:
Any reason for not using `find_by`? It seems better to return the Item, and not the an `Array`